docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode.
+
+## Install the toolchain
+
+First, you need the ONE toolchain. Let's install the ONE toolchain. (You should install the in-house backend to follow this.)
+
+<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=600 />
+
+## Prepare a NN model and a config file
+
+Second, you need a NN model and a config file to run compiling. Now you can get one from [res/modelDir/truediv](../res/modelDir/truediv/). Add it to your workspace. Now you can see a model and a config.
+
+<img src="https://user-images.githubusercontent.com/10216715/174798969-eee44fea-bd71-4e6a-8e2c-9e1de37ad74a.gif" width=600 />
+
+## Compile models
+
+Last, you can compile the model by pushing Run button.
```

Comment:
```suggestion
Last, you can compile the model by pushing Run button.

Depends on backend, you may be able to infer or profile a model by 'running config'.
For the details, please find (**a link to the guide of onecc**)
```

Reply:
(**a link to the guide of onecc**) <- ONE_USER_GUIDE, maybe?

Reply:
Hmm... how about this link?
- https://github.com/Samsung/ONE/tree/master/compiler/one-cmds

Now I cannot find a guide for onecc

Reply:
https://pages.github.sec.samsung.net/AIP/ONE_User_Guide

This one :-D It has example of one cfg.

Reply:
The link is connecting to the private server.

Reply:
Oh.... Sure!

Reply:
```suggestion
Last, you can compile the model by pushing Run button.

Depends on backend, you may be able to infer or profile a model by 'running config'.
```

If there is no link for the official guide, it would be better to skip it.